### PR TITLE
Next bundle release 2022 12 06

### PR DESCRIPTION
**Pull request by Temppus**

### CODE REVIEW

File: sf-docker/special/sf-with-keycloak/sf-server/.env [hunk 1]
```diff
@@ -49,6 +49,6 @@ REGISTRY=registry.gitlab.com/innovatrics/smartface/
 
 # Version
 SF_VERSION=v5_4.16.0
```
Comment (marek-nereca) [inline sf-docker/special/sf-with-keycloak/sf-server/.env:51]: wrong version

File: sf-docker/multi-camera/.env [hunk 2]
```diff
@@ -45,7 +45,7 @@ GstPipelineTemplate_Jetson=uridecodebin uri={0} source::latency=0 ! queue max-si
 REGISTRY=registry.gitlab.com/innovatrics/smartface/
 
 # Version
-SF_VERSION=v5_4.15.0
-AC_VERSION=v5_1.7.0
+SF_VERSION=v5_4.17.0
+AC_VERSION=v5_1.8.0
 SFS_VERSION=v5_1.12.0
```
Comment (marek-nereca) [inline sf-docker/multi-camera/.env:50]: 1.14

**Comment by marek-nereca** [comment]
Dont forget that we do not have sf-watchlist-api image. We should use sf-api and set FeatureManagement__Watchlist env variable OR create sf-watchlist-api image again to not break old deploments. 
FYI: @blaskoa

File: sf-docker/cloud-matcher/docker-compose.yml [hunk 3]
```diff
@@ -80,6 +80,17 @@ services:
       - S3Bucket__BucketName
       - S3Bucket__AccessKey
       - S3Bucket__SecretKey
+      - NoSqlDataStorageDisabled
+      - Authentication__UseAuthentication
```
Comment (blaskoa) [inline sf-docker/cloud-matcher/docker-compose.yml:84]: do we need to forward these `Authentication` and `Multitenancy` envs?

**Review by blaskoa — APPROVED** [review]

**Pull request merged**